radio/src/gui/colorlcd/radio_setup.cpp

```diff
@@ -404,7 +404,12 @@ class AlarmsPage : public Page {
       new StaticText(line, rect_t{}, STR_INACTIVITYALARM, 0, COLOR_THEME_PRIMARY1);
       edit = new NumberEdit(line, rect_t{}, 0, 250, GET_SET_DEFAULT(g_eeGeneral.inactivityTimer));
       lv_obj_set_style_grid_cell_x_align(edit->getLvObj(), LV_GRID_ALIGN_STRETCH, 0);
-      edit->setSuffix("minutes");
+      std::string suffix(STR_MINUTE_PLURAL2);
+      if(g_eeGeneral.inactivityTimer < g_min_plural2)
+        suffix = std::string(STR_MINUTE_SINGULAR);
+      else if(g_eeGeneral.inactivityTimer < g_max_plural2)
```

Comment:
I don't know how it work in other slavic languages but in polish this case is more complicated:
x2 - x4: PLURAL2, where x == 0 || x >= 2
i.e.
2,3,4: PLURAL2
**12,13,14: PLURAL1**
22,23,24: PLURAL2
32,33,34: PLURAL2
and so on